src/reporter.ts

```diff
@@ -471,12 +472,19 @@ export class RPReporter implements Reporter {
   processAnnotations({ annotations, test }: { annotations: Annotation[]; test?: TestCase }): void {
     annotations.forEach(({ type, description }) => {
       if (type && description) {
-        const data = JSON.parse(description);
-        const reportData = {
-          type,
-          data,
-        };
-        this.onEventReport(reportData, test);
+        try {
+          const data = safeParse(description);
```

Comment:
Does it make sense to try parsing annotation in case it is corresponding only to the supported RP events?